prober/http.go

```diff
@@ -424,6 +426,60 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 		body = body_file
 	}
 
+	// If a multipart form is configured, add it to the request.
```

Comment:
this implementation and configuration feels a little awkward, can we make it simple and implement this with just one bool flag called `multipart`?

If the flag is set, you can just make the check multi-part. for text and data, you can use the existing `Body` and `BodyFile` config values?

this would make it easy to configure and simplify this implementation as well?